ouroboros-network-framework/ouroboros-network-framework.cabal

```diff
@@ -76,15 +76,11 @@ library
                      , contra-tracer
 
                      , io-classes      >=0.1   && < 0.3
-                     , monoidal-synchronisation
-                                       >=0.1   && < 0.2
```

Comment:
Could you leave it, it's used in `p2p-master` branch which sooner or later will be merged to `master`.

Reply:
Brought back, thanks!